db/migrations/20190715125635_add_commentable_to_comment.cr

```diff
@@ -0,0 +1,15 @@
+class AddCommentableToComment::V20190715125635 < Avram::Migrator::Migration::V1
+  def migrate
+    alter :comments do
+      add commentable_type : String?
+      add commentable_id : Int64?
```

Comment:
I think it would be nice to have an `add_polymorphic_belongs_to :commentable` that generates these for you. It could take a `foreign_key_type` option that defaults to `Int64`. Also would be great to specify optional or not with `optional: true`

Reply:
added! 😄